stellar-contract-env-host/src/host.rs

```diff
@@ -177,6 +177,27 @@ struct RollbackPoint {
     objects: usize,
 }
 
+#[cfg(feature = "testutils")]
+pub trait ContractVTableFnTrait {
```

Comment:
I understand what a vtable, but I don't think it should be a concept exposed by the SDK. What does a `ContractVTableFnTrait` represent? Does it represent a function that can be called with a list of arguments?
```suggestion
pub trait ContractFn {
```

Reply:
I was able to replace with `Fn(&[RawVal]) -> RawVal` (note: _not_ `Fn(RawVal) -> RawVal` because it must accept a variable number of parameters), although it took some work. The trait no longer exists so no name to change.